src/styles/blocks/card.scss

```diff
@@ -0,0 +1,130 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat($column-count, 200px);
```

Comment:
The variable `$column-count` is used here but is not defined outside of media queries. This will cause an error because SCSS variables are not scoped to media queries. Consider defining `$column-count` outside of the media queries and updating its value within each media query.

Reply:
the variable is defined in a separate file ‘./utils/variables’